guides/NEW_APP.md

```diff
@@ -0,0 +1,50 @@
+Creating a new app
+==================
+
+The following steps will guide you through add adding a new app to our system that is built by the CI system and deployed to the live environment.
+
+Prerequisites
+-------------
+
+In order to follow this guide you will need:
+
+* A github account with access to the ONSdigital organisation
+
+* A login for the concourse CI environment
+
+* [dp-cli](https://github.com/ONSdigital/dp-cli)
+
+* Git:
+
+  ```
+  brew install git
+  ```
+
+* Terraform >= 0.12:
+
+  ```
+  brew install terraform
+  ```
+
+* Fly cli (can be downloaded from the links in the corner of the concourse UI)
```

Comment:
```suggestion
* Fly CLI (can be downloaded from the links in the corner of the concourse UI)
```
Initialism in capitals (concourse refers to it in caps too, https://concourse-ci.org/fly.html )